src/khoj/utils/helpers.py

```diff
@@ -282,6 +282,13 @@ class ConversationCommand(str, Enum):
     ConversationCommand.Help: "Display a help message with all available commands and other metadata.",
 }
 
+tool_descriptions_for_llm = {
+    ConversationCommand.Default: "Use this if there might be a mix of general and personal knowledge in the question",
+    ConversationCommand.General: "Use this when you can answer the question without any outside information or personal knowledge",
```

Comment:
Minor suggestion to improve the general command. Unsure if it's better
```suggestion
    ConversationCommand.General: "Use this when you can answer the question without needing any additional online or personal information",
```